data_and_models/pipelines/sentence_embedding/train.py

```diff
@@ -77,8 +77,8 @@ def main():
         )
         model._stop_words_id = 0
         print("Saving model to disk...")
-        with open(out_dir / "model.pkl", "wb") as f:
-            joblib.dump(model, f)
+        with (out_dir / "model.pkl").open("wb") as f:
+            pickle.dump(model, f)
```

Comment:
Just out of curiosity, is this significantly slower than using `joblib` or it makes no difference?

Reply:
I tested it and in our use case it seems that `pickle` is actually faster!

Possible explanation [here](https://stackoverflow.com/a/12617603):
> if you don't pickle large numpy arrays, then regular pickle can be significantly faster, especially on large collections of small python objects (e.g. a large dict of str objects) because the pickle module of the standard library is implemented in C while joblib is pure python.